packages/ui/src/lib/settingsNavItem/SettingsNavItem.svelte

```diff
@@ -66,7 +69,12 @@ function click(): void {
     class:hover:border-[var(--pd-secondary-nav-text-hover-bg)]={!selected}>
     <span class="group-hover:block flex flex-row items-center" class:capitalize={!child}>
       {#if icon}
-        <Fa class="mr-4" icon={icon} />
+        {#if isFontAwesomeIcon(icon)}
+          <Fa class="mr-4" {icon} />
+        {:else}
+          <!-- svelte-ignore svelte_component_deprecated -->
```

Comment:
Hummm, can't we avoid that ?

Reply:
I was able to make it work only with the <svelte:component> and because it is deprecated, it needs to have this 
If I try something like <icon size.. class..></icon> I think that the Component is not rendered (assuming that the test is not passing)